ServerCore/Pages/Puzzles/Checklist.cshtml.cs

```diff
@@ -83,8 +110,12 @@ public async Task<IActionResult> OnGetAsync()
                     Prerequisites = prereqs != null ? string.Join(", ", prereqs) : "",
```

Comment:
It would be good if the prereqs were sorted here. Very noticeable when two puzzles have the same prereqs but they appear different.